src/modules/commander/Commander.cpp

```diff
@@ -703,6 +703,18 @@ Commander::Commander() :
 	_param_mav_type = param_find("MAV_TYPE");
 	_param_rc_map_fltmode = param_find("RC_MAP_FLTMODE");
 
+	int32_t value_int32 = 0;
+
+	// MAV_SYS_ID => vehicle_status.system_id
+	if ((_param_mav_sys_id != PARAM_INVALID) && (param_get(_param_mav_sys_id, &value_int32) == PX4_OK)) {
```

Comment:
Get rid of `_param_mav_sys_id` in the header, we can just do it locally and not store anything.

```suggestion
	param_t param_mav_sys_id = param_find("MAV_SYS_ID");
	if ((param_mav_sys_id != PARAM_INVALID) && (param_get(param_mav_sys_id, &value_int32) == PX4_OK)) {
```